concepticondata/conceptlists/RonaTas-2011-431.tsv

```diff
@@ -0,0 +1,432 @@
+RonaTas-2011-431-24	24	governor of Croatia, governor of the southern marches		
+RonaTas-2011-431-25	25	to regret, to be sorry	1875	REGRET
+RonaTas-2011-431-26	26	hairy, bushy (of dogs); a big hairy dog (as noun)		
+RonaTas-2011-431-27	27	label, tag		
+RonaTas-2011-431-28	28	cattle, livestock	1153	CATTLE
+RonaTas-2011-431-29	29	velvet		
+RonaTas-2011-431-30	30	to fuck		
+RonaTas-2011-431-31	31	courageous, brave, valiant	3	BRAVE
+RonaTas-2011-431-32	32	to estimate, to esteem, to appreciate		
+RonaTas-2011-431-33	33	frog	503	FROG
+RonaTas-2011-431-34	34	hobble, shackle, leg-iron		
+RonaTas-2011-431-35	35	stamp, bond, mark	1720	SIGN
+RonaTas-2011-431-36	36	wage, rent		
+RonaTas-2011-431-37	37	name of a people		
+RonaTas-2011-431-38	38	grain or small globule of preciousmetal	605	GRAIN
+RonaTas-2011-431-39	39	a bird of prey		
+RonaTas-2011-431-40	40	Pecheneg, name of a tribe		
+RonaTas-2011-431-41	41	letter (of the alphabet)	1612	LETTER
+RonaTas-2011-431-42	42	pocket knife	3742	POCKETKNIFE
+RonaTas-2011-431-43	43	bull	1008	BULL
+RonaTas-2011-431-44	44	shackles		
+RonaTas-2011-431-45	45	judge	861	JUDGE
+RonaTas-2011-431-46	46	to be forgiven		
+RonaTas-2011-431-47	47	to forgive	1528	FORGIVE
+RonaTas-2011-431-48	48	buckler with handle, handle of different arms		
+RonaTas-2011-431-49	49	knot, bend	760	KNOT
+RonaTas-2011-431-50	50	stack of hay		
+RonaTas-2011-431-51	51	tassel, fringe, tuft, node	2989	FRINGE
+RonaTas-2011-431-52	52	burdock, Arctium sp.		
+RonaTas-2011-431-53	53	ankle	803	ANKLE
+RonaTas-2011-431-54	54	ankle boot		
+RonaTas-2011-431-55	55	anthill	1873	ANTHILL
+RonaTas-2011-431-56	56	wine	1524	WINE
+RonaTas-2011-431-57	57	to cover, to overturn	761	COVER
+RonaTas-2011-431-58	58	cover	761	COVER
+RonaTas-2011-431-59	59	calf	1989	CALF
+RonaTas-2011-431-60	60	pepper	1385	PEPPER
+RonaTas-2011-431-61	61	pea, Pisum sativum; bean	832	BEAN
+RonaTas-2011-431-62	62	badger, Meles meles	3717	BADGER
+RonaTas-2011-431-63	63	witch, sorceress	824	WITCH
+RonaTas-2011-431-64	64	to annoy	2892	MAKE ANGRY (ANNOY)
+RonaTas-2011-431-65	65	revenge	495	REVENGE
+RonaTas-2011-431-66	66	stick, cane	1295	STICK
+RonaTas-2011-431-67	67	a kind of beer, alcoholic beverage	404	ALCOHOL (FERMENTED DRINK)
+RonaTas-2011-431-68	68	rich; abundant, roomy	712	RICH
+RonaTas-2011-431-69	69	horsefly, gadfly	3262	HORSEFLY
+RonaTas-2011-431-70	70	fast(ing)	1631	FAST
+RonaTas-2011-431-71	71	wise	698	WISE
+RonaTas-2011-431-72	72	cradle	2946	CRADLE
+RonaTas-2011-431-73	73	bison	3142	BISON
+RonaTas-2011-431-74	74	prison, jail	1281	PRISON
+RonaTas-2011-431-75	75	Ismaelite, name of an ethnic group in Medieval Hungary		
+RonaTas-2011-431-76	76	sad, bad-tempered	699	SAD
+RonaTas-2011-431-77	77	farewell, indulgence, dedication festival, annual parish-feast	3554	GOODBYE
+RonaTas-2011-431-78	78	ox; also ox, goat, ram without or with small or unusual horns;stupid	1502	GOAT
```

Comment:
I'd prefer to map to `1169 OX`, as this seems to be the principle meaning

Reply:
This seems to denote a concept like ANIMAL WITH HORNS so it is too broad to be mapped to OX.

Reply:
From the original source: "The H[ungarian] word is well represented in the H[ungarian] dial[ect]s of Central and Western Hungary, but the first attestation as a common noun is fixed relatively late, in 1775, on a wordlist of the slang of a certain thief (Schram 1962: 108). As a PN it first occurs in 1215 (copy 1550) and later (Fehértói 2004: 157)"

I'll leave it unmapped, since it's a dialectal word. Would need to check Fehértói 2004: 157 to read more about the meaning

Reply:
Resolved in 98a7e57a41cbe42bd46bc878ec2850518c4988b5